<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWareHouseItemsBatchItems</class>
 <widget class="QDialog" name="DialogWareHouseItemsBatchItems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new batch item</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="barcodeLabel">
       <property name="text">
        <string>Barcode:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="BarcodeLineEdit" name="barcodeLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="amountLabel">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="amountDoubleSpinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="descriptionTextEdit">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/images/list-remove.svg</normaloff>:/images/list-remove.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Del</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="pushButtonSave">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/images/document-save.svg</normaloff>:/images/document-save.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="pushButtonAdNew">
     <property name="text">
      <string>Add new batch item</string>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/images/list-add.svg</normaloff>:/images/list-add.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+N</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarcodeLineEdit</class>
   <extends>QLineEdit</extends>
   <header>barcodelineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>barcodeLineEdit</tabstop>
  <tabstop>amountDoubleSpinBox</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>pushButtonAdNew</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
